pages/views/_layouts/base.njk

```diff
@@ -22,7 +22,28 @@
     <link rel="stylesheet" type="text/css" href="{{ '/bundles/localdev.css' | url }}" />
   {% endblock %}
   {% block head %}
-    <script async src="https://cse.google.com/cse.js?cx=3171f866738b34f02"></script>
+    <script async>
```

Comment:
Just a small note: the inline script can not be async.
`async` is a first of all attribute for load process adjustments, inline script doesn't load it is already on the page.